devsetup/scripts/gen-edpm-compute-node.sh

```diff
@@ -186,10 +186,12 @@ if [ ! -f ${DISK_FILEPATH} ]; then
     fi
     qemu-img create -o backing_file=${CRC_POOL}/centos-9-stream-base.qcow2,backing_fmt=qcow2 -f qcow2 "${DISK_FILEPATH}" "${EDPM_COMPUTE_DISK_SIZE}G"
     if [[ ! -e /usr/bin/virt-customize ]]; then
-        if [[ $(awk '{print $6}' /etc/redhat-release) =~ ^8.* ]]; then
+        if [[ $(awk '{print $6}' /etc/redhat-release) =~ ^8.* ]] || \
```

Comment:
Maybe we can use this to simplify `rpm -E %{rhel}`. 

[fultonj@hamfast zed]$ if [[ $(rpm -E %{rhel}) == 8 ]]; then echo 1; fi
1
[fultonj@hamfast zed]$ 

[fultonj@osp-storage-01 ~]$ if [[ $(rpm -E %{rhel}) == 9 ]]; then echo 1; fi
1
[fultonj@osp-storage-01 ~]$

Reply:
This command works on centos 9 too.
```
[root@edpm-compute-0 ~]# rpm -E %{rhel}
9
[root@edpm-compute-0 ~]# cat /etc/redhat-release 
CentOS Stream release 9
[root@edpm-compute-0 ~]# 
```